CMakeLists.txt

```diff
@@ -156,9 +160,11 @@ if(NOT HAVE_TRILINOS OR NOT USE_TRILINOS_EXODUS)
 endif()
 
 include_directories(src)
-include_directories(src/mpi_buckets/dist_hash)
```

Comment:
please just remove these lines referencing mpi_buckets (and the include in contact_manager)
this mpi_buckets is dead code

Reply:
Done.